Image fails to build

Error while installing `lmfit` dependency for `nexpy`

```
2022/09/28 09:51:10 packer-builder-qemu plugin: [DEBUG] Opening new ssh session                                                                            
2022/09/28 09:51:10 packer-builder-qemu plugin: [DEBUG] starting remote command: chmod +x /tmp/script_2378.sh; PACKER_BUILDER_TYPE='qemu' PACKER_BUILD_NAME
='qemu' PACKER_HTTP_ADDR='10.0.2.2:8488' PACKER_HTTP_IP='10.0.2.2' PACKER_HTTP_PORT='8488'  /tmp/script_2378.sh                                            
    qemu: Collecting nexpy                                                                                                                                 
    qemu:   Downloading https://files.pythonhosted.org/packages/c6/66/95d2fde6877d5abfcc846082faab557009992010e15255bd660a433b5cc2/NeXpy-0.13.3-py3-none-an
y.whl (658kB)                                                                
    qemu: Requirement already satisfied: scipy in /usr/lib/python3/dist-packages (from nexpy)                                                              
    qemu: Requirement already satisfied: ipython in /usr/lib/python3/dist-packages (from nexpy)                                                            
    qemu: Collecting nexusformat>=0.6.1 (from nexpy)                                                                                                       
    qemu:   Downloading https://files.pythonhosted.org/packages/5f/8f/652b07d0302ea6ece79658e4fb9fd2aa3006631d03a5e52db08a4501d72d/nexusformat-0.7.7-py3-no
ne-any.whl (80kB)                                                                                                                                          
    qemu: Requirement already satisfied: matplotlib in /usr/lib/python3/dist-packages (from nexpy)                                                         
    qemu: Collecting lmfit>=1.0.2 (from nexpy)                                                                                                             
    qemu:   Downloading https://files.pythonhosted.org/packages/8c/2f/8af90d45e585692eca1922c53e239c7aa97a904d9254df5b08f2f32520ac/lmfit-1.0.3.tar.gz (292k
B)                                                                                                                                                         
    qemu:     Complete output from command python setup.py egg_info:                                                                                       
    qemu:     Traceback (most recent call last):                         
    qemu:       File "<string>", line 1, in <module>                     
    qemu:       File "/tmp/pip-build-chkg1dl0/lmfit/setup.py", line 55, in <module>                                                                        
    qemu:         packages=['lmfit'],                                                                                                                      
    qemu:       File "/usr/lib/python3/dist-packages/setuptools/__init__.py", line 129, in setup                                                           
    qemu:         return distutils.core.setup(**attrs)                                                                                                     
    qemu:       File "/usr/lib/python3.6/distutils/core.py", line 108, in setup                                                                            
    qemu:         _setup_distribution = dist = klass(attrs)                                                                                                
    qemu:       File "/usr/lib/python3/dist-packages/setuptools/dist.py", line 372, in __init__                                                            
    qemu:         _Distribution.__init__(self, attrs)              
    qemu:       File "/usr/lib/python3.6/distutils/dist.py", line 281, in __init__                                                                         
    qemu:         self.finalize_options()                                                                                                                  
    qemu:       File "/usr/lib/python3/dist-packages/setuptools/dist.py", line 528, in finalize_options                                                    
    qemu:         ep.load()(self, ep.name, value)                                                                                                          
    qemu:       File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2324, in load                                                        
    qemu:         return self.resolve()                                                                                                                    
    qemu:       File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2330, in resolve                                                     
    qemu:         module = __import__(self.module_name, fromlist=['__name__'], level=0)                                                                    
    qemu:       File "/tmp/pip-build-chkg1dl0/lmfit/.eggs/setuptools_scm-7.0.5-py3.6.egg/setuptools_scm/__init__.py", line 5                               
    qemu:         from __future__ import annotations                                                                                                       
==> qemu: Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-chkg1dl0/lmfit/                                                    
    qemu:         ^                                                                                                                                        
    qemu:     SyntaxError: future feature annotations is not defined                                                                                       
    qemu:                                                                                                                                                  
    qemu:     ----------------------------------------                    
2022/09/28 09:51:15 packer-builder-qemu plugin: [ERROR] Remote command exited with '1': chmod +x /tmp/script_2378.sh; PACKER_BUILDER_TYPE='qemu' PACKER_BUI
LD_NAME='qemu' PACKER_HTTP_ADDR='10.0.2.2:8488' PACKER_HTTP_IP='10.0.2.2' PACKER_HTTP_PORT='8488'  /tmp/script_2378.sh
```

--- a/templates/visa-apps/applications/nexpy/scripts/install.sh
+++ b/templates/visa-apps/applications/nexpy/scripts/install.sh
@@ -1,2 +1,2 @@
 #!/bin/bash
-pip3 install nexpy
+pip3 install nexpy==0.10.12

